testing/run_tests.py

```diff
@@ -490,6 +482,8 @@ def make_test(name, flags=None, extra_env=None):
         'MTL_SHADER_VALIDATION_TEXTURE_USAGE':
             '1',  # Validates that texture references are not nil.
         'VK_ICD_FILENAMES': os.path.join(build_dir, 'vk_swiftshader_icd.json'),
+        'VK_LAYER_PATH': os.path.join(build_dir, 'vulkan-data'),
```

Comment:
A comment pointing to the GN file with the string that has to be the same as this one would be good?

Reply:
done